tests/integration/blockchain/test_blockchain.py

```diff
@@ -613,6 +613,14 @@ async def test_multi_block_file_sync(self):
 
 
 class TestGeneralBlockchainSync(SyncingBlockchainTestCase):
+    async def test_sync_waits_for_lbrycrd_to_start(self):
+        await self.sync.stop()
+        await self.chain.stop()
+        sync_start = asyncio.ensure_future(self.sync.start())
```

Comment:
Should this be `create_task()` instead?

according to python docs create_task is preferred: https://docs.python.org/3.7/library/asyncio-future.html#asyncio.ensure_future

Reply:
nice catch, pushed a change